lib/Partitioner/PartitionerUtils.cpp

```diff
@@ -17,10 +17,66 @@
 
 #include "glow/Partitioner/PartitionerUtils.h"
 
-#include <set>
+using llvm::isa;
 
 namespace glow {
 
+/// The nodes in function \p F which be grouped into levels based on how far
+/// they are from the roots.
+BFSLevel getBFSLevel(Function *F) {
+  // Visit graph nodes in BFS order. For each non-storage node, get its level.
```

Comment:
I understand that this function was just moved around and there we no/minimal changes to it. But could you add comments before each loop describing at the high-level what it does?